services/EventService.ts

```diff
@@ -15,11 +15,19 @@ export default class EventService {
     this.transactions = new TransactionsManager(entityManager);
   }
 
-  public async create(event: Event) {
+  /**
+   * Creates a new event
+   *
+   * @param event object with all the properties of the event
+   * @returns The event that was created
+   */
+  public async create(event: Event): Promise<PublicEvent> {
     const eventCreated = await this.transactions.readWrite(async (txn) => {
       const eventRepository = Repositories.event(txn);
       const isUnusedAttendanceCode = eventRepository.isUnusedAttendanceCode(event.attendanceCode);
       if (!isUnusedAttendanceCode) throw new UserError('Attendance code has already been used');
+      const isStartAfterEnd = event.start > event.end;
+      if (isStartAfterEnd) throw new UserError('Start date after end date');
```

Comment:
Actually lets follow Shravan's advice as the error is self explanatory enough that the boolean is somewhat redundant